environment.yml

```diff
@@ -23,7 +23,6 @@ dependencies:
     - black
     - blackdoc
     - docformatter
-    - flake8
```

Comment:
TODO wait for https://github.com/conda-forge/staged-recipes/pull/18471 to be merged.